### DISCUSSION TITLE
IRC-31 implementation

### DISCUSSION THREAD

**Discussion by quocle108** [icon-project/btp#4] [Q&A] [answered]
Hello,

We're working on Fee Aggregation Java SCORE, and we're looking for IRC-31 implementation for testing purpose. But there are no token IRC-31 [at java library](https://github.com/sink772/javaee-tokens/tree/master/tokens/src/main/java/com/iconloop/score/token)

Can you support us about this?

**Comment by cow-hs** [comment]
we have only pyee contract https://github.com/sink772/multi-token-standard

**Reply by flowersinthesand** [reply]
Could you let us know when it can be available?

**Reply by cow-hs** [reply]
There is currently no Java implementation schedule due to internal schedule.

**Comment by flowersinthesand** [comment, marked as answer]
ICON NativeCoinBSH will substitute for IRC 31 implementation
